### DISCUSSION TITLE
Can I lock a pdf with password?

### DISCUSSION THREAD

**Discussion by bobaikato** [simonhaenisch/md-to-pdf#95] [Q&A] [answered]
We have a use-case where we want to generate a PDF from HTML or MD, but we want it protected with a password.

**Comment by simonhaenisch** [comment, marked as answer]
It's not something that I could easily implement because there's no API for it in Chromium. However you can use something like `qpdf` to do this as a post-compilation step. There's a node wrapper for it: https://www.npmjs.com/package/node-qpdf.

**Reply by bobaikato** [reply]
Thanks.
